openapi.yaml

```diff
@@ -3400,11 +3400,11 @@ components:
           example: 2
```

Comment:
Other schema validation updates will need to be reflected in openapi.yaml as well. It can be done in a future PR.

Reply:
Updated